.github/ISSUE_TEMPLATE/1_bug.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report.
+about: You are creating a Game with Flame but you are noticing strange behaviour, throws an exception, or it is buggy.
```

Comment:
nit: noticing some strange behaviour, that it throws an unexpected exception, or that it is not working according to the specifications.